ruby/2.3-admin/Dockerfile

```diff
@@ -8,13 +8,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
     libsqlite3-dev \
     libxml2-dev \
     libxslt-dev \
-    make && \
-  apt-get -y purge \
-    keyboard-configuration \
-    libruby1.9.1 \
-    perl \
-    ruby1.9.1 \
-    ruby2.0 \
```

Comment:
I misspoke yesterday. Clortho uses ruby2.0 and the sdk. Some day it should be rewritten in the python sdk (boto3) so we can stop installing it :)